backends-clickhouse/src/test/scala/org/apache/gluten/execution/GlutenClickHouseMergeTreeWriteOnS3Suite.scala

```diff
@@ -635,6 +671,7 @@ class GlutenClickHouseMergeTreeWriteOnS3Suite
 
     withSQLConf(
       "spark.databricks.delta.optimize.minFileSize" -> "200000000",
+      "spark.gluten.sql.columnar.backend.ch.runtime_settings.mergetree.insert_without_local_storage" -> "true",
```

Comment:
why add this config

Reply:
Test directly merge parts on s3. See this code branch
![image](https://github.com/user-attachments/assets/26c868db-1880-457e-9100-ac6cd987ea76)